addons/product/migrations/13.0.1.2/openupgrade_analysis_work.txt

```diff
@@ -83,7 +83,7 @@ product      / product.template.attribute.line / product_template_value_ids (man
 product      / product.template.attribute.line / product_template_value_ids (many2many): not a function anymore
 product      / product.template.attribute.line / product_template_value_ids (many2many): type is now 'one2many' ('many2many')
 product      / product.template.attribute.value / attribute_line_id (many2one)  : NEW relation: product.template.attribute.line, required
-# DONE: post-migration: filled field (using the same approach than in OU v12)
+# DONE: pre-migration: filled required field (using the same approach than in OU v12)
```

Comment:
If I don't remember bad, if you this, there are missing values that are fill lately.

Reply:
I initially moved that to pre-migration because the field attribute_line_id of ptav is required. Later, it proves useful in pre-migration, in correctly filling the missing ptavs [the query of missing ptavs use that field in order to make it compatible with the sql_constraint unique(attribute_line_id, product_attribute_value_id)]